README.markdown

```diff
@@ -464,7 +464,11 @@ manifests as a valid password attribute. *Type*: rvalue
 * `to_bytes`: Converts the argument into bytes, for example 4 kB becomes 4096.
 Takes a single string value as an argument. *Type*: rvalue
 
-* `type`: Returns the type when passed a variable. Type can be a string, array, hash, float, integer, or boolean. *Type*: rvalue
+* `type3x`: Returns a string description of the type when passed a variable. Type can be a string, array, hash, float, integer, or boolean. This function will be removed when puppet 3 support is dropped and the new type system may be used. *Type*: rvalue
+
+* `type_of`: Returns the literal type when passed a variable. Requires the new
+  parser. The output is meant for human use; for programatic inspection of types
```

Comment:
While it is true that interpolating the resulting type (or passing it to notice) will produce a string representation of the type, it is also very useful to get the type of something and using that result multiple times (as opposed to doing multiple =~ where the type has to be inferred each time.) Note that `$value =~ Numeric` is the same as doing `type_of($value) <= Numeric`.